Update validator-auctions-bonds-and-rewards.mdx (#98)
chainflip-io/chainflip-docs: pages/concepts/validator-network/validator-auctions-bonds-and-rewards.mdx

### What is the Validator Auction?

The Validator slots are granted based on the results of regular Auctions, whereby the highest bidders that complete Keygen at the end of each Auction are selected as the Authority Set for the next Epoch and a Rotation is conducted to handover Authority from one set to the next. On Perseverance, the time between each is usually 3 hours long but will be longer in production.

Bidding is done from a Validator's State Chain account, which is funded through the [State Chain FLIP Gateway](state-chain-flip-gateway.md).

Auctions begin halfway between the start and the expected end of an Epoch. During Auctions:

- **Bidding Validators** commit their FLIP balance to the auction for its duration and may not change their state to Non-Bidding.

> - **Non-Bidding Validators** (without a current bond) can redeem funds as normal, but can't change their state to bidding during the auction

Reviewer: ```suggestion
- **Non-Bidding Validators** (without a current bond) can redeem funds as normal, and *can* change their state to bidding at any time, including during the auction
```
